docs/src/solutions.md

```diff
@@ -116,6 +117,84 @@ end
 # TODO: How to accurately measure the solve time.
 ```
 
+## Sensitivity analysis
+
+Given an LP problem and an optimal solution corresponding to a basis, we can
+question how much an objective coefficient or standard form rhs coefficient
+(c.f., [`standard_form_rhs`](@ref)) can change without violating primal or dual
+feasibility of the basic solution.
+
+Given an LP optimal solution (and both [`has_values`](@ref) and
+[`has_duals`](@ref) returns `true`) [`lp_objective_perturbation_range`](@ref)
```

Comment:
Also mention that you must use a solver that computes and returns the optimal basis. Many of the solvers available in JuMP for solving LPs either don't compute an optimal basis at all or don't yet have this exposed.